pkg/webhook/policy.go

```diff
@@ -242,6 +242,23 @@ func (h *validationHandler) getValidationMessages(res []*rtypes.Result, req *adm
 	}
 	for _, r := range res {
 		if err := util.ValidateEnforcementAction(util.EnforcementAction(r.EnforcementAction)); err != nil {
+			log.WithValues(
```

Comment:
We may not want to always log unknown enforcement actions. It could be an action that simply isn't relevant for this EP.

Let's defer this since it will be affected by the pending enforcement action revamp anyway.